circle.yml

```diff
@@ -5,3 +8,13 @@ dependencies:
     - "target/streams"
     - "project/target/resolution-cache"
     - "project/target/streams"
+  override:
+    - sbt test:compile
+test:
+  override:
+    - sbt test
+  post:
+    - sbt riffRaffUpload
```

Comment:
IIRC `post` actions in the test section might happen regardless of the test success. For flex I solved this by doing the upload in the deploy section. See https://github.com/guardian/flexible-content/blob/master/circle.yml#L69-L73